docs/source/libraries/xgboost.rst

```diff
@@ -5,15 +5,21 @@ XGBoost
 
 XGBoost_ is a popular Gradient Boosting library with Python interface.
 eli5 supports :func:`eli5.explain_weights` and :func:`eli5.explain_prediction`
-for XGBClassifer_ and XGBRegressor_ estimators. It is tested for
+for XGBClassifer_, XGBRegressor_ and Booster_ estimators. It is tested for
 xgboost >= 0.6a2.
 
 .. _XGBoost: https://github.com/dmlc/xgboost
 .. _XGBClassifer: https://xgboost.readthedocs.io/en/latest/python/python_api.html#xgboost.XGBClassifier
 .. _XGBRegressor: https://xgboost.readthedocs.io/en/latest/python/python_api.html#xgboost.XGBRegressor
+.. _Booster: http://xgboost.readthedocs.io/en/latest/python/python_api.html#xgboost.Booster
 
-:func:`eli5.explain_weights` uses feature importances. Additional
-arguments for XGBClassifer_ and XGBRegressor_:
+:func:`eli5.explain_weights` uses feature importances.
+There is one additional required argument for Booster_ estimator:
+
+* ``is_regression`` True if solving a regression problem
```

Comment:
@kmike I think I'm happy with solution in dc1b405:
* ``is_regression`` is removed from ``explain_weights`` arguments, cause we don't need it - so it has the same API as for other xgboost estimators
* for ``explain_prediction``, ``is_regression`` is optional. It it is not set and booster has just one target, we assume regression - in this case, proba would not be shown. I think it's less bad to not show proba then to show a wrong proba, and I would not like to compare values of score and proba to decided if proba should be shows. For a multitarget classifier, is_regression is set to False if it was not passed (because there is no multitarget regression in xgboost).